docs/KNOWNISSUES.md

```diff
@@ -2,138 +2,119 @@
+The `#!` magic number may be added to a script that doesn't have a `.ps1` extension, but this will cause the script to be run in a new PowerShell instance preventing the script from working properly when interchanging objects.
+(Note: this may be the desirable behavior when executing a PowerShell script from `bash` or another shell.)
 
 ### Missing command aliases
 
-On Linux/macOS, the "convenience aliases" for the basic commands `ls`, `cp`,
-`mv`, `rm`, `cat`, `man`, `mount`, `ps` have been removed.
-On Windows, PowerShell provides a set of aliases that map to Linux command
-names for user convenience.
-These aliases have been removed from the default PowerShell on Linux/macOS distributions,
-allowing the native executable to be run instead.
-There are pros and cons to having do this.
-It exposes the native command experience to the PowerShell user but reduces
-functionality in the shell because the native commands return strings not objects.
+On Linux/macOS, the "convenience aliases" for the basic commands `ls`, `cp`, `mv`, `rm`, `cat`, `man`, `mount`, `ps` have been removed.
+On Windows, PowerShell provides a set of aliases that map to Linux command names for user convenience.
+These aliases have been removed from the default PowerShell on Linux/macOS distributions, allowing the native executable to be run instead.
+
+There are pros and cons to doing this.
+Removing the aliases exposes the native command experience to the PowerShell user, but reduces functionality in the shell because the native commands return strings instead of objects.
 
 > NOTE: This is an area where the PowerShell team is looking for feedback.
-> What is the preferred solution? Should we leave it as is or add the
-> convenience aliases back? See
+> What is the preferred solution? Should we leave it as is or add the convenience aliases back? See
 > [Issue #929](https://github.com/PowerShell/PowerShell/issues/929).
 
 ### Missing Wildcard (globbing) Support
 
-Currently, PowerShell only does wildcard expansion (globbing) for the
-built-ins but not for external commands.
-This means that a command like `ls *.txt` will fail because the asterisk will not be
-expanded to match file names.
-You can work around this by doing `ls (gci *.txt | % name)` or, more simply,
-`gci *.txt` using the PowerShell built-in equivalent to `ls`.
-[RFC0009](https://github.com/PowerShell/PowerShell-RFC/issues/33)
+Currently, PowerShell only does wildcard expansion (globbing) for built-in cmdlets, but not for external commands or binaries.
+This means that a command like `ls *.txt` will fail because the asterisk will not be expanded to match file names.
+You can work around this by doing `ls (gci *.txt | % name)` or, more simply, `gci *.txt` using the PowerShell built-in equivalent to `ls`.
+
+See [RFC0009](https://github.com/PowerShell/PowerShell-RFC/issues/33) to give us feedback on how to improve the globbing experience on Linux/macOS.
```

Comment:
That RFC is now closed.  Perhaps we can reference https://github.com/PowerShell/PowerShell/issues/954 instead which I expect to be partially implemented by beta1 at least supporting *.txt type scenarios.